auditbeat/_meta/fields.common.yml

```diff
@@ -19,6 +19,7 @@
       description: Set if the file has the `setgid` bit set. Omitted otherwise.
 
     - name: origin
+      # Should this be keyword?
```

Comment:
`file.origin` is only filled on macOS. It can be a URL or a computer name, maybe more. We treat URLs as keywords in other places, so I think keyword would  make sense here, too.

/cc @adriansr who wrote the code for this

Reply:
This should be a keyword

Reply:
@adriansr Could you open a PR for this change?

Reply:
Strings should always be `keyword`. `text` should be added as a multi-field if deemed necessary.

When in doubt, `keyword`